src/biogeochem/CNPhenologyMod.F90

```diff
@@ -972,6 +1034,147 @@ subroutine CNSeasonDecidPhenology (num_soilp, filter_soilp       , &
  
   end subroutine CNSeasonDecidPhenology
 
+  !-----------------------------------------------------------------------
+  function SeasonalCriticalDaylength( g, p ) result( crit_daylat )
+    !
+    ! !DESCRIPTION:
+    ! Function to determine the critical day length needed for seasonal
+    ! decidious leaf offset. When depends on latitude it's higher for
+    ! high latitudes and lower for temperate regions.
+    !
+    ! !ARGUMENTS:
+    integer, intent(IN) :: g ! Gridcell index
+    integer, intent(IN) :: p ! Patch index
+    real(r8) :: crit_daylat  ! Return value
+    !
+    ! !LOCAL VARIABLES:
+    !-----------------------------------------------------------------------
+
+    select case( critical_daylight_method )
+    ! Critical day length depends on both vegetation type and latitude
+    case(critical_daylight_depends_on_latnveg)
+        ! Critical day length for offset is fixed for temperate type vegetation
+        if ( pftcon%season_decid_temperate(patch%itype(p)) == 1 )then
+           crit_daylat = crit_dayl
+        ! For Arctic vegetation -- critical daylength is higher at high latitudes and shorter
+        ! at midlatitudes
+        else
```

Comment:
wait, what happen to this formula when you're north of 65 degrees?  It makes crit_daylat > crit_dayl_at_high_lat.  Is that what's @lmbirch89 intended? Should `crit_daylat` be more than 15 hr (54000 seconds)?

Reply:
That is what's currently being done. In talking with Leah we thought we should stick with that.